fabric-common/lib/EventListener.js

```diff
@@ -110,8 +101,7 @@ class EventListener {
 	 * event information or error instance.
 	 * @param {Error} error - An Error object that was created as a result
 	 *  of an error on the {@link EventService} connection to the Peer.
-	 * @param {BlockEvent} event - A {@link BlockEvent} that contains
-	 *  event information.
+	 * @param {EventInfo} event - A {@link EventInfo} that contains event information.
 	 */
 	onEvent(error, event) {
```

Comment:
This looks to me like it should be private too. It's only intended to be called internally to do the invocation of the application-supplied callback function